lib/Peer.js

```diff
@@ -20,7 +20,8 @@ class Peer extends Events {
   }
 
   transport (dest) {
-    return this.newTransport({ dest: dest })
+    const opts = Object.assign({}, this.conf, { dest: dest })
```

Comment:
@robertkowalski can we use lodash `extend` or `defaults` (it's just to keep things uniform)